### ISSUE TITLE
When will the training codes be available

### ISSUE THREAD

**Issue by may0324** [TencentARC/VTLayout#1]
Hello , I reimplemented this work based on the code of layoutlmv3, but the training loss was not converged. I thought there must be something not aligned.
When will the training codes be available?
